src/doc/trpl/hello-cargo.md

```diff
@@ -32,6 +32,13 @@ $ mkdir src
 $ mv main.rs src/main.rs
 ```
 
+Note that since we're creating an executable, we used `main.rs`. If we
+want to make a compiled library instead, we  should use `lib.rs`.
```

Comment:
Maybe just 'library' instead of 'compiled library'? Also, there's two spaces between 'we' and 'should'